server/src/main/java/com/exacaster/lighter/rest/SessionController.java

```diff
@@ -105,4 +105,15 @@ public Statement postStatements(@PathVariable String id, @PathVariable String st
     public Statement cancelStatements(@PathVariable String id, @PathVariable String statementId) {
         return sessionService.cancelStatement(id, statementId);
     }
+
+    @Post("/statements")
+    @Status(HttpStatus.CREATED)
+    public Statement postStatements(@Valid @Body Statement statement) {
```

Comment:
Again, those should be managed through the same endpoints, as regular session statements, we might want to have permanent sessions on jupyter notebooks too.